roles/pulp/defaults/main.yml

```diff
@@ -1,22 +1,30 @@
 ---
-
-pulp_cache_dir: '/var/lib/pulp/tmp'
-pulp_config_dir: '/etc/pulp'
+pulp_cache_dir: "/var/lib/pulp/tmp"
+pulp_config_dir: "/etc/pulp"
 # This var intentionally not advertised to users, because there is no
 # foreseeable need for them to change the filename, only pulp_config_dir.
-pulp_settings_file: '{{ pulp_config_dir }}/settings.py'
-pulp_default_admin_password: ''
-pulp_install_dir: '/usr/local/lib/pulp'
+pulp_settings_file: "{{ pulp_config_dir }}/settings.py"
+pulp_default_admin_password: ""
+# Users should not set this variable, instead using `pulp_settings.databases`
+pulp_settings_db_defaults:
```

Comment:
The role's README.md needs an update about the loose coupling between pulp and pulp-database. See pulp-content/README.md's "Shared variables"

Reply:
Update still needed.

Reply:
it is already there:
https://github.com/pulp/ansible-pulp/blob/master/roles/pulp/README.md

Reply:
I dislike the wording, but I suppose it is good enough.